WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -383,7 +382,6 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
 private extension ProductFormViewController {
     func configureNavigationBar() {
         updateNavigationBar(isUpdateEnabled: false)
-        updateNavigationBarTitle(productName: product.name)
```

Comment:
nit: `updateNavigationBarTitle` function can also be deleted now that it's not used anymore?

Reply:
Good catch, removed here 540c20b893c98ab5a16643dceab1a6a1f95d1471 💯